Tella/Domain/Entity/Uwazi/CollectedTemplate.swift

```diff
@@ -72,6 +76,14 @@ class CollectedTemplate: Codable, Hashable {
                 throw DecodingError.dataCorruptedError(forKey: .entityRow, in: container, debugDescription: "entityRow issue")
             }
         }
+        
+        if let relationshipsString = try container.decodeIfPresent(String.self, forKey: .relationships) {
+            if let jsonData = relationshipsString.data(using: .utf8) {
```

Comment:
I think it's better to use `String` `extension` to have the decoded relationships